modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_publisher/render/list/entry/view.jsp

```diff
@@ -127,6 +127,14 @@ CPCatalogEntry cpCatalogEntry = cpContentHelper.getCPCatalogEntry(request);
 						</div>
 					</div>
 
+					<div class="autofit-col">
```

Comment:
is this the right position in the end?

Reply:
Talked with design team, and now the position is good.